package/MDAnalysis/lib/log.py

```diff
@@ -80,12 +80,11 @@
 .. autogenerated, see Online Docs
 
 """
-from __future__ import print_function, division, absolute_import
+from __future__ import absolute_import
 
```

Comment:
Remove?

Reply:
Sure, I'll remove all the py2 stuff from lib at the same time.